.travis.yml

```diff
@@ -1,22 +1,23 @@
-language: node_js
+language: rust
+dist: focal
```

Comment:
The version of GDK that is shipped still seems too old for the Travis build to succeed, I don't know if you have any idea of how it could be fixed?